infra-templates/portworx/1/rancher-compose.yml

```diff
@@ -0,0 +1,56 @@
+.catalog:
+  name: "Portworx"
+  version: "1.2.0"
+  description: "Portworx Docker Volume Plugin"
+  questions:
+
+    - variable: "VOLUME_DRIVER_NAME"
+      description: "Name to use for this plugin instance with Docker:  docker volume create -d <volume driver name>"
+      label: "Volume Driver Name"
+      required: true
+      default: "pxd"
+      type: "string"
+    - variable: "CLUSTER_ID"
+      description: "Arbitrary Cluster ID, common to all nodes in PX cluster "
+      label: "Cluster ID"
+      type: "string"
+      required: true
+      default: ""
+    - variable: "KVDB"
+      description: "Key-value database, accessibe to all nodes in PX cluster (Ex: 'etcd://10.0.0.42:4001')"
+      label: "Key-Value Database"
+      type: "string"
+      required: true
+      default: ""
+    - variable: "USE_DISKS"
+      description: "Cmdline args for disks to use. Ex: '-a' for all available, or '-s /dev/sdX' for each individual disk"
+      label: "Use Disks"
+      type: "string"
+      required: true
+      default: "-s /dev/xvdb"
+    - variable: "HEADER_DIR"
+      description: "Directory where kernel headers can be found.  Default is '/usr/src'.  For CoreOS use '/lib/modules'"
+      label: "Headers Directory"
+      type: "string"
+      required: true
+      default: "/usr/src"
+
+
+portworx:
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.hostname_override: container_name
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
+    io.rancher.container.pull_image: always
```

Comment:
Don't needed labels here, also added at docker-compose.
```
-  labels:
-    io.rancher.container.dns: 'true'
-    io.rancher.container.hostname_override: container_name
-    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
-    io.rancher.container.pull_image: always
```

Reply:
got it. i will validate it, untill then let it be as it is for now.

Reply:
Please, review and validate it before approve.

Reply:
Sure, Who needs to review and validate? is it something within portworx?

Reply:
Please remove labels from rancher-compose, already added to docker-compose

Reply:
Please remove labels from rancher-compose, already added to docker-compose

Reply:
removed labels from rancher-compose.